service/src/main/java/org/mskcc/cmo/metadb/service/impl/CorrectCmoPatientHandlingServiceImpl.java

```diff
@@ -159,6 +162,15 @@ public void run() {
                         if (expectedCount != samplesAfterSwap.size()) {
                             LOG.error("Expected sample count after patient ID swap does not match actual"
                                     + " count: " + expectedCount + " != " + samplesAfterSwap.size());
+                        } else {
+                            // expectedCount == samplesAfterSwap.size()
```

Comment:
let's add a log saying that we're publishing the metadata history for [sample id]

we log something similar in the sample update handler